CONTRIBUTING.md

```diff
@@ -1,5 +1,29 @@
-# Bug Reports & Feature Requests
+# List of rules and norms
+
+## Bug Reports & Feature Requests
 Bug reports and feature requests are greatly appreciated and can be done in form of an [issue](https://github.com/JakeRoggenbuck/auto-clock-speed/issues)
 
-# Develop
+## Develop
 If you would like to help complete a task, comment in the issue and we may assign it to you!
+
+## Pull Requests
+- Pull requests must be tested but with `cargo test` and ran on local machine (laptop with battery)
+- Pull requests that change core functionality must have a review
+
+## Issues marked with `help wanted` or `good first issue`
+- Issues marked with either `help wanted` or `good first issue` are a great place to start!
+- Issues marked with these are a great place for discussion and user suggestions
+
+## Projects tab
+- The [projects tab](https://github.com/JakeRoggenbuck/auto-clock-speed/projects/1) is something we use extensively to organize when things get done
+
```

Comment:
Just an idea, but maybe we should start thinking of a wiki to put all the documentations into?

Reply:
yee